packages/gasket-plugin-command/lib/index.d.ts

```diff
@@ -99,7 +99,7 @@ export function processCommand(command: GasketCommandDefinition): GasketCommand;
 declare module '@gasket/core' {
 
   export interface HookExecTypes {
-
+    build(): any;
```

Comment:
Left as `any` due to the fact that I'm not 100% on the build hook signature's in the internal repo.